packages/core/src/Resource/Change/Task/Helpers/findCreateTasksForResources.js

```diff
@@ -0,0 +1,14 @@
+import { resourceIsNew } from '../../Inspection';
+import findTaskByPayload from './findTaskByPayload';
+
+export default function findCreateTasksForResources(tasks, resources) {
+  if (!resources) {
+    return [];
+  }
+
+  const newResources = resources.filter(resource => resourceIsNew(resource));
+
+  return newResources
+    .map(resource => findTaskByPayload(tasks, resource))
+    .filter(relationCreateTask => typeof relationCreateTask !== 'undefined');
```

Comment:
Je zou denk ik ook de relationCreateTask direct kunnen teruggeven omdat de waarde faulty is. 
Dit is echter wel duidelijker.

Reply:
Ik had het eerst opgesplist toen het nog in een grote functie zat. Inmiddels is alles opgesplistst en is het wat mij betreft niet meer nodig.